### ISSUE TITLE
Kanban-Board-React - Information

### ISSUE THREAD

**Issue by rudicarrilloypr** [rudicarrilloypr/space-travelers-react#25]
### **2 Members:**
- Rodolfo carrillo Garcia
- Nafeur Rahman Adil

### **Link to project:**
https://github.com/rudicarrilloypr/space-travelers-react

**Comment by Shedrack-Sunday** [comment]
Hi @rudicarrilloypr  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Invalid Code Review Request 🔴 

You have submitted a project, that does not link to the right project link. Kindly add the right project link,

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Shedrack-Sunday** [comment]
Hi @rudicarrilloypr  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Invalid Code Review Request 🔴 

Just as I said previously, you can't proceed unless you have the cards assigned to each member. If your partner is not responding, kindly reach out to student success in this regard. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._
